sdk/src/main/java/com/silanis/esl/sdk/examples/AccountRolesExample.java

```diff
@@ -0,0 +1,49 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.api.model.AccountRole;
+import com.silanis.esl.api.model.Result;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public class AccountRolesExample extends SDKSample {
+    Result<AccountRole> result = null;
+    Result<AccountRole> result2 = null;
+    Result<AccountRole> result3 = null;
+    String newAccountRoleId = null;
+    AccountRole newAccountRole = null;
+    List<String> newAccountUsers = null;
+
+    public static void main(String... args) {
+        new AccountRolesExample().run();
+    }
+
+    public void execute() {
+        result = eslClient.getAccountService().getAccountRoles();
+        String newAccountRoleName = UUID.randomUUID().toString();
+        AccountRole accountRole = new AccountRole().setName(newAccountRoleName)
```

Comment:
We should not use the API model directly in the examples. You need to create an AccountRole model for the SDK and use the builder pattern. Then add converters for the API model and SDK model.
And you need to add unit tests for all converters and builders you newly created.

Reply:
Fixed